flink-formats/flink-json/src/main/java/org/apache/flink/formats/json/JsonRowDataDeserializationSchema.java

```diff
@@ -85,16 +90,22 @@ public JsonRowDataDeserializationSchema(
                 new JsonToRowDataConverters(failOnMissingField, ignoreParseErrors, timestampFormat)
                         .createConverter(checkNotNull(rowType));
         this.timestampFormat = timestampFormat;
+        this.allowUnescapedControlChars = allowUnescapedControlChars;
         boolean hasDecimalType =
                 LogicalTypeChecks.hasNested(rowType, t -> t instanceof DecimalType);
         if (hasDecimalType) {
             objectMapper.enable(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS);
         }
+        objectMapper.configure(
+                JsonReadFeature.ALLOW_UNESCAPED_CONTROL_CHARS.mappedFeature(),
+                allowUnescapedControlChars);
     }
 
     @Override
     public RowData deserialize(byte[] message) throws IOException {
         try {
+            // it will return null if `ignoreParseErrors` is true
+            // and when executing `objectMapper.readTree(message)` throw an exception.
```

Comment:
This comment is hard to understand. Why exception will be thrown when `ignoreParseErrors` is enabled.